src/Main.java

```diff
@@ -1,7 +1,65 @@
+import java.util.*;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        findOccurrence(toList(new String[]{"Sun", "Moon", "Cat", "Dog", "House", "Book", "Car", "House", "Friend",
+                "Love", "Music", "Cat", "Sky", "Moon", "Cat", "Love", "Sky", "Phone", "Dog", "Dream"}));
+
+
+    }
+
+    public static int countOccurance(ArrayList<String> list, String word) {
+        int count = 0;
+        for (String s : list) {
+            if (s.equals(word)) {
+                count++;
+            }
+        }
+        return count;
+    }
+
+    public static <T> ArrayList<T> toList(T[] arr) {
```

Comment:
<T> ArrayList<T> need to be ArrayList<Integer> according to the task

Reply:
Don`t understand...